sdv/constraints/utils.py

```diff
@@ -104,3 +104,59 @@ def sigmoid(data, low, high):
     data = data * (high - low) + low
 
     return data
+
+
+def check_nans_row(row):
```

Comment:
minor: can we rename this to `get_nan_component_value`, since this is getting the value to go in the `nan_component` column?